src/less/components/sidebar.less

```diff
@@ -0,0 +1,14 @@
+#new-file-input {
+  box-shadow: 0 0 0 1600px rgba(0, 0, 0, 0.10);
+  border-radius: 0;
+  border-left: 4px solid white;
+  font-size: 14px;
+}
+
+.add-file-input .bp3-tree-node-label {
+  overflow: visible;
+}
+
+.bp3-tree-node-caret-none {
+  min-width: 8px;
+}
```

Comment:
On a new startup, the sidebar takes up enough space that a lot of the editor text is wrapped.

What would you think of some combo of narrowing the sidebar / making the whole window wider?

Reply:
Addressed in c39632f

made the size `1400` and the sidebar tile % smaller.